src/common/geometry.js

```diff
@@ -128,17 +128,36 @@ export const getSliderBounds = (vertices, sliderValue) => {
 }
 
 // returns an array of points drawing a circle of a given radius
-export const circle = (radius=1, start=0) => {
+export const circle = (radius, start=0, x=0, y=0) => {
   let points = []
 
-  for (let i=start; i<=128+start; i++) {
-    let angle = Math.PI * 2.0 / 128.0 * i
-    points.push(new Victor(Math.cos(angle)*radius, Math.sin(angle)*radius))
+  for (let i=0; i<=128; i++) {
+    let angle = Math.PI * 2.0 / 128.0 * i + start
+    points.push(new Victor(x + Math.cos(angle)*radius, y + Math.sin(angle)*radius))
   }
 
   return points
 }
 
+export const arc = (radius, startAngle, endAngle, x=0, y=0) => {
+  let resolution = (Math.PI*2.0) / 128.0 // 128 segments per circle. Enough?
```

Comment:
If I had infinite time, I might change this to be at least some minimum number, but determine it based on the radius. So a small circle might have 16 sides, but a large one might have 256.